packages/node/vro-language-server/src/server/request/collection/WorkspaceCollection.ts

```diff
@@ -52,28 +66,163 @@ export class WorkspaceCollection {
         }
     }
 
-    private async triggerCollectionAndRefresh(workspaceFolder: WorkspaceFolder): Promise<void> {
+    async triggerCollectionAndRefresh(workspaceFolder: WorkspaceFolder): Promise<void> {
         await this.triggerCollection(workspaceFolder)
         this.hints.refreshForWorkspace(workspaceFolder)
     }
 
     private async triggerCollection(workspaceFolder: WorkspaceFolder): Promise<void> {
+        this.logger.info("Triggering workspace collection...")
         const modules: string[] = []
         const outputDir = this.environment.resolveHintsDir(workspaceFolder)
 
         this.getModulesForCollection(workspaceFolder, undefined, modules)
         if (modules.length === 0) {
-            this.logger.debug("Skipping workspace collection as there are no JS action modules in the project")
+            this.logger.info("Skipping workspace collection as there are no JS action modules in the project")
             return
         }
 
-        const command = `mvn o11n-hint:collect -DoutputHintDirectory="${outputDir}" -pl ${modules.join(",")} -e`
-        const cmdOptions = { cwd: workspaceFolder.uri.fsPath }
+        const fullPath = path.join(workspaceFolder.uri.fsPath, modules.join(","))
+        const modulesPath = path.join(fullPath, "src/main/resources")
```

Comment:
I guess this should work for both JS and TS projects, the path for TS projects is only src/?